docs/sdks/swift-sdk/integrate-with-auth0.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Integrate With Auth0
+sidebar_position: 3
+---
+
+This guide describes how to configure Auth0 to delegate to Beyond Identity for authentication during an OAuth2 authorization flow. 
+
+### Prerequisites
+
+ - [Integrate With Auth0](/guides/sso-integrations/integrate-with-auth0)
+ - [Swift SDK Overview](overview)
+
+Before calling [`Embedded.shared.authenticate`](overview#authentication), we must [Authorize With Auth0](#authorize-with-auth0).
+
+### Authorize With Auth0
+
+ - Step 1: Configuring the Authenticator Config
+
+Make sure the [Authentication Config](/docs/v1/platform-overview/authenticator-config#embedded) in the Beyond Identity Console is set to type `Embedded` and that the Invoke URL points to your application with either an App Scheme or a Universal Link.
+
+ - Step 2: Auth0 Authorize URL
+
+To begin the authentication flow, start an `ASWebAuthenticationSession`, and load the OAuth2 authorization request URL provided by Okta. Make sure `prefersEphemeralWebBrowserSession` is set to `false` as the browser will need to share cookies or other browsing data between the authentication session.
+
+```javascript
+let session = ASWebAuthenticationSession(
+    url: viewModel.auth0URL,
+    callbackURLScheme: viewModel.callbackScheme
+    completionHandler: { (url, error) in }
+)
+session.prefersEphemeralWebBrowserSession = false
```

Comment:
Should we mention the differences between setting this to true/false?

Reply:
This is written above the example. Do you want to expand on it?

> Make sure `prefersEphemeralWebBrowserSession` is set to `false` as the browser will need to share cookies or other browsing data between the authentication session.

Reply:
I would probably change "will need" to "may need". We don't required cookies but Okta/Auth0 may.

Reply:
It seems necessary in this case. Okta works in both cases but Auth0 doesn't work without setting this to false.